Libplanet/Store/LiteDBStore.cs

```diff
@@ -426,20 +426,26 @@ public override AddressStateMap GetBlockStates(HashDigest<SHA256> blockHash)
         /// <inheritdoc/>
         public override IEnumerable<Tuple<HashDigest<SHA256>, long>> IterateStateReferences(
             Guid chainId,
-            Address address)
+            Address address,
+            long? highestIndex,
+            long? lowestIndex,
+            int? limit)
         {
+            highestIndex = highestIndex ?? long.MaxValue;
+            lowestIndex = lowestIndex ?? 0;
+
             string collId = StateRefId(chainId);
             LiteCollection<StateRefDoc> coll = _db.GetCollection<StateRefDoc>(collId);
             string addressString = address.ToHex().ToLower();
             IEnumerable<StateRefDoc> stateRefs = coll.Find(
                 Query.And(
+                    Query.All("BlockIndex", Query.Descending),
                     Query.EQ("AddressString", addressString),
-                    Query.All("BlockIndex", Query.Descending)
-                )
+                    Query.Between("BlockIndex", lowestIndex, highestIndex)
```

Comment:
Does it work even if `lowestIndex` or `highestIndex` is null?

Reply:
If the parameters are null, it will be changed to `0` or `long.MaxValue` [here](https://github.com/planetarium/libplanet/pull/545/files#diff-3fe3c469228f579007de2d2703d077d9R434).